README.md

```diff
@@ -44,8 +44,21 @@ $ brew install dnsmasq
 $ sudo brew services start dnsmasq
 ```
 
+### What services might you have available? Let's find out! ###
```

Comment:
I'm 99.9% sure this *is* in fact what's available. Reason is, I didn't even know if `brew services list` existed until I straight up tried it to find out, and it listed all the "services" I'd installed in the past. For example, Cassandra, MySQL, PostgreSQL, rethinkdb. Never used it to start any of those before (since I just found out about this project).